flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClusterDescriptorTest.java

```diff
@@ -124,13 +144,19 @@ public void testKillCluster() throws Exception {
 		descriptor.deploySessionCluster(clusterSpecification);
 
 		final KubernetesClient kubeClient = server.getClient();
-		assertEquals(2, kubeClient.services().list().getItems().size());
+
```

Comment:
I am not sure why could not get the rest svc here. Since we just mock the get request, not the post.

Reply:
Actually we mock the request of **ANY** kind, not only the **GET**.